pkg/controller/customresource/customresource.go

```diff
@@ -148,6 +148,9 @@ func SetAnnotation(resource mdbv1.AtlasCustomResource, key, value string) {
 	resource.SetAnnotations(annot)
 }
 
+// IsGov verify the given domain is Atlas for Government
+// Deprecated: This function was move to another package.
+// Start using github.com/mongodb/mongodb-atlas-kubernetes/pkg/controller/atlas instead
 func IsGov(domain string) bool {
```

Comment:
Who is using this interface apart from us?

Reply:
Nobody, it's more of an alert if we need to add a new check for any resource